packages/test_track_test_support/lib/src/fake_test_track.dart

```diff
@@ -6,14 +6,22 @@ import 'package:test_track/test_track.dart';
 /// of the call site performing the `ab`, and the
 /// third parameter is the nullable name of the
 ///  default [Variant]
-typedef Ab = bool Function(String, String, String?);
+typedef Ab = bool Function(
+  String splitName,
+  String context,
+  String? trueVariant,
+);
```

Comment:
I found that without these names, the tab completion in vs code gave me sad names like p0, p1, p2 for the lambda. This makes it inherit the good names.

Reply:
Oh, thanks for doing this. Since these are explicit now, can we remove the redundant documentation on each typedef that details the types of the lambda arguments?

Reply:
good call